populus/compilation/backends/solc_combined_json.py

```diff
@@ -29,7 +28,7 @@
     ContractsNotFound,
 )
 
-from populus.utils.compile import (
+from ..helpers import (
```

Comment:
There *may* be other places on the source which use `from ..` style imports but I've tended to try and avoid anything beyond one dot and to prefer full paths in these cases.  One dot is pretty easy to reason about since it's importing something from the same level as the current module but once we start reaching up through the tree I find it hard to reason about.

Can you change this to a full path?